geode-redis/README.md

```diff
@@ -200,16 +200,16 @@ start server \
 |                    	| SINTER                                             	| CLUSTER INFO                  	|
 |                    	| SINTERSTORE                                        	| CLUSTER KEYSLOT                 	|
 |                    	| SISMEMBER                                          	| CLUSTER MEET                     	|
-|                    	| SMOVE                                              	| CLUSTER MYID                   	|
-|                    	| SPOP                                               	| CLUSTER NODES                 	|
-|                    	| SRANDMEMBER                                        	| CLUSTER REPLICAS                 	|
-|                    	| SSCAN                                              	| CLUSTER REPLICATE              	|
-|                    	| STRLEN                                             	| CLUSTER RESET                  	|
-|                    	| SUNION                                             	| CLUSTER SAVECONFIG                |
-|                    	| SUNIONSTORE                                        	| CLUSTER SET-CONFIG-EPOCH          |
-|                    	| TIME                                               	| CLUSTER SETSLOT                 	|
-|                    	| UNLINK [1]                                         	| CLUSTER SLAVES                  	|
-|                    	|                                                    	| CLUSTER SLOTS                 	|
+|                    	| SLOWLOG                                            	| CLUSTER MYID                   	|
```

Comment:
Good call adding this to the README! We also need to add it to the list of unsupported commands in this file: `redis_api_for_geode.html.md.erb`

Reply:
Done, thanks!